luxonis_ml/data/loader.py

```diff
@@ -91,6 +91,13 @@ def __init__(
                 synced_classes = json.load(file)
             for task in synced_classes:
                 self.dataset.set_classes(classes=synced_classes[task], task=task)
+            skeletons_file = os.path.join(self.dataset.metadata_path, "skeletons.json")
+            try:
+                with open(skeletons_file, "r") as file:
+                    synced_skeletons = json.load(file)
+                self.dataset.set_skeletons(synced_skeletons)
+            except FileNotFoundError:
```

Comment:
If there is an existing remote dataset which doesn't have this file (made before these changes) will the loading still work or will it hit this error?

Reply:
After testing the loading of the detection COCO validation dataset from the _mlcloud-dev_ _luxonis_train_project_ example, it initially failed because `self.fs.get_file("metadata/skeletons.json", local_file)` does not function without a specified file. However, I resolved this issue by moving the statement to a point after verifying the file's existence. Currently, I'm running the easyml remote command for this COCO detection example  to ensure it functions correctly. It's still in the pending state, but I will keep you updated on its progress.